packages/framework/src/Component/Cron/CronModuleFacade.php

```diff
@@ -206,7 +216,11 @@ public function getCronModuleByServiceId(string $serviceId): CronModule
      */
     public function findAllIndexedByServiceId(): array
     {
-        return $this->cronModuleRepository->findAllIndexedByServiceId();
+        if ($this->allIndexedByServiceId === null) {
```

Comment:
If the service uses the internal cache, it should implement the `ResetInterface`.

Reply:
Good catch!